wordpress-mobile/WordPress-iOS
# Stats: Add File Downloads support to the Period store

Ref #11963 
Related WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/174

To test:
Since File Downloads isn't actually displayed yet, just verify the query is being run.
- Go to any Period. Verify `Stats: Finished fetching file downloads.` appears in the logs.
- Refresh Periods. Verify `Stats: Finished fetching file downloads.` appears in the logs again.
- Bonus: to verify WPKit is returning something, you can log out the `downloads` parameter in the `receivedFileDownloads` method. It should look:
```
Optional(WordPressKit.StatsFileDownloadsTimeIntervalData(period: __C.StatsPeriodUnit, periodEndDate: 2019-07-29 06:00:00 +0000, totalDownloadsCount: 0, otherDownloadsCount: 0, fileDownloads: []))
```

Update release notes:
